setup_workspace.sh

```diff
@@ -0,0 +1,23 @@
+# This file is used for Bazel CI to install rust and setup the workspace.
+
+# install rust
+apt-get update && apt-get install curl -y && curl https://sh.rustup.rs -sSf | sh -s -- -y
+
+# move workspace file to parent dir
+mv test_suite/WORKSPACE WORKSPACE
+
+# replace cargo in workspace with the absolute path $HOME/.cargo/bin/cargo
+sed -i -e 's/cargo/\$HOME\/.cargo\/bin\/cargo/g' WORKSPACE
+
+# create an empty build file
+touch BUILD
+
+# modify the testdata_path
+cat > test_suite/testenv.py <<EOF
+STARLARK_BINARY_PATH = {
+    "java": "external/io_bazel/src/main/java/com/google/devtools/starlark/Starlark",
+    "go": "external/net_starlark_go/cmd/starlark/darwin_amd64_stripped/starlark",
+    "rust": "external/starlark-rust/target/debug/starlark-repl",
+}
+STARLARK_TESTDATA_PATH = "test_suite/"
+EOF
```

Comment:
Why do you need to rewrite the file?
Oh, is it just because the TESTDATA_PATH changes when you move the WORKSPACE file? That's another reason to avoid doing it. :)